datajunction-server/datajunction_server/api/data.py

```diff
@@ -34,11 +34,11 @@
 from datajunction_server.service_clients import QueryServiceClient
 from datajunction_server.utils import get_query_service_client, get_session
 
-router = APIRouter()
+router = APIRouter(tags=["data"])
 
 
-@router.post("/data/{node_name}/availability/")
-def add_an_availability_state(
+@router.post("/data/{node_name}/availability/", name="Add Availability State To Node")
```

Comment:
I would also de-capitalize `To` but feel free to leave it for later, since you did a lot of work in this PR already.